packages/core/database/encryption/encryption-schema-registry.js

```diff
@@ -109,6 +118,71 @@ function registerCustomSchema(schema) {
     );
 }
 
+/**
+ * Loads and registers encryption schemas from API module definitions.
+ * Each module can declare credentialFields to encrypt in its encryption config.
+ *
+ * @param {Array} integrations - Array of integration classes with modules
+ */
+function loadModuleEncryptionSchemas(integrations) {
+    if (!integrations || !Array.isArray(integrations)) {
+        return;
+    }
+
+    const moduleSchemas = {};
+
+    for (const Integration of integrations) {
+        const integrationDef = Integration?.Definition;
+        if (!integrationDef || !integrationDef.modules) {
+            continue;
```

Comment:
Shouldn't this throw an error? As an Integration can not live without a definition.